src/SpotifyLogin.jsx

```diff
@@ -0,0 +1,16 @@
+import React from "react"
+
+export const SpotifyLogin = () => {
+    const initiateSpotifyLogin = () => {
+        const authUrl =
+            `https://accounts.spotify.com/authorize?` +
+            `response_type=token` +
+            `&client_id=962be0d99c5f4098985705f22c488288` +
+            `&redirect_uri=http://localhost:1234/auth` +
```

Comment:
It's not gonna work when you deploy your code to the real hosting. Don't hardcode localhost, but dynamically get current hostname instead